share/spice/indego_bikeshare_locations/bike_sharing_indego_phl.css

```diff
@@ -0,0 +1,32 @@
+.tile--indego_phl {
```

Comment:
rename file to `indego_bikeshare_locations.css`, and all `indego_phl` occurrences in this file to `indego_bikeshare_locations`